iotdb-operator/config/samples/iotdb_v1_datanode.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: iotdb.apache.org/v1
+kind: DataNode
+metadata:
+  name: iotdb-datanode
+spec:
+  image: apache/iotdb:1.3.2-datanode
+  replicas: 2
```

Comment:
it might be more reasonable to let the whole sample deploy a standalone IoTDB cluster. then the datanode replicas could be 1 and the datanode pv2 could be ignored.